libzeth/core/joinsplit.hpp

```diff
@@ -2,11 +2,11 @@
 //
 // SPDX-License-Identifier: LGPL-3.0+
 
-#ifndef __ZETH_TYPES_JOINSPLIT_HPP__
-#define __ZETH_TYPES_JOINSPLIT_HPP__
+#ifndef __ZETH_CORE_JOINSPLIT_HPP__
```

Comment:
I'd be tempted to rename this file `joinsplit_input` as there is a mismatch between the name of the file and the class here